rmgpy/rmg/main.py

```diff
@@ -431,14 +433,9 @@ def initialize(self, **kwargs):
         self.logHeader()
         
         try:
-            restart = kwargs['restart']
+            self.restart_seed_path = kwargs['restart']
```

Comment:
I think this doesn't do anything. It should set the `coreSeedPath`, `edgeSeedPath`, `filtersPath`, and `speciesMapPath` like you do when loading the input file.

Alternatively, if you don't think we should support both methods, you can remove this and set the `-r` flag to simply print a warning that the old restart method has been removed and the flag does not do anything.

Reply:
Fixed, thanks for the catch!